### ISSUE TITLE
nextTick is not available when building with Rollup

### ISSUE THREAD

**Issue by lahdekorpi** [shershen08/vue-masonry#114]
Rollup build fails due to nextTick missing.

```
➜  masonry git:(master) ✗ npm run build

> masonry@0.0.0 build
> vue-tsc --noEmit && vite build

vite v2.2.4 building for production...
✓ 91 modules transformed.
'nextTick' is not exported by node_modules/vue-masonry/node_modules/vue/dist/vue.runtime.esm.js, imported by node_modules/vue-masonry/src/masonry-vue3.plugin.js
file: /Users/toni/Documents/Omat/masonry/node_modules/vue-masonry/src/masonry-vue3.plugin.js:5:9
3:
4: // Vue 3 Global API changed: nextTick import
5: import { nextTick } from 'vue';
            ^
6:
7: const attributesMap = {
error during build:
Error: 'nextTick' is not exported by node_modules/vue-masonry/node_modules/vue/dist/vue.runtime.esm.js, imported by node_modules/vue-masonry/src/masonry-vue3.plugin.js
    at error (/Users/toni/Documents/Omat/masonry/node_modules/rollup/dist/shared/rollup.js:5309:30)
    at Module.error (/Users/toni/Documents/Omat/masonry/node_modules/rollup/dist/shared/rollup.js:9765:16)
    at Module.traceVariable (/Users/toni/Documents/Omat/masonry/node_modules/rollup/dist/shared/rollup.js:10151:29)
    at ModuleScope.findVariable (/Users/toni/Documents/Omat/masonry/node_modules/rollup/dist/shared/rollup.js:8905:39)
    at ChildScope.findVariable (/Users/toni/Documents/Omat/masonry/node_modules/rollup/dist/shared/rollup.js:2636:38)
    at ClassBodyScope.findVariable (/Users/toni/Documents/Omat/masonry/node_modules/rollup/dist/shared/rollup.js:2636:38)
    at FunctionScope.findVariable (/Users/toni/Documents/Omat/masonry/node_modules/rollup/dist/shared/rollup.js:2636:38)
    at ChildScope.findVariable (/Users/toni/Documents/Omat/masonry/node_modules/rollup/dist/shared/rollup.js:2636:38)
    at FunctionScope.findVariable (/Users/toni/Documents/Omat/masonry/node_modules/rollup/dist/shared/rollup.js:2636:38)
    at ChildScope.findVariable (/Users/toni/Documents/Omat/masonry/node_modules/rollup/dist/shared/rollup.js:2636:38)
```

```
├── @vitejs/plugin-vue@1.2.2
├── @vue/compiler-sfc@3.0.11
├── autoprefixer@10.2.5
├── axios@0.21.1
├── mitt@2.1.0
├── postcss@8.2.14
├── typescript@4.2.4
├── vite@2.2.4
├── vue-class-component@8.0.0-rc.1
├── vue-masonry@0.13.0
├── vue-router@4.0.6
├── vue-tsc@0.1.2
├── vue@3.0.11
└── vuex@4.0.0
```

**Comment by lahdekorpi** [comment]
Oddly enough, with Vite, everything works perfectly when running just the dev server.

**Comment by 7architect** [comment]
i have a similar problem with **Vite** and **Vue 3**. I tried to build with npm, yarn, pnpm and have a problem with nextTick import 

![image](https://user-images.githubusercontent.com/33001830/118513601-2c0a7f80-b734-11eb-8ee8-bf77a8d8d3a0.png)

<details>
  <summary>Text logs</summary>
  
 ```
➜ pnpm build 

> dbuff-vite@0.0.0 build /home/alexander/code/dbuff-vite
> vite build

vite v2.2.4 building for production...
transforming (94) node_modules/.pnpm/@iconicicons+vue3@1.1.1_vue@3.0.11/node_modules/@iconicicons/vue3/icons/esm/ArrowDownLeftIcon.js
warn - You have enabled the JIT engine which is currently in preview.
warn - Preview features are not covered by semver, may introduce breaking changes, and can change at any time.
✓ 368 modules transformed.
'nextTick' is not exported by node_modules/.pnpm/vue@2.6.12/node_modules/vue/dist/vue.runtime.esm.js, imported by node_modules/.pnpm/vue-masonry@0.13.0/node_modules/vue
-masonry/src/masonry-vue3.plugin.js
file: /home/alexander/code/dbuff-vite/node_modules/.pnpm/vue-masonry@0.13.0/node_modules/vue-masonry/src/masonry-vue3.plugin.js:5:9
3: 
4: // Vue 3 Global API changed: nextTick import
5: import { nextTick } from 'vue';
            ^
6: 
7: const attributesMap = {
error during build:
Error: 'nextTick' is not exported by node_modules/.pnpm/vue@2.6.12/node_modules/vue/dist/vue.runtime.esm.js, imported by node_modules/.pnpm/vue-masonry@0.13.0/node_modu
les/vue-masonry/src/masonry-vue3.plugin.js
    at error (/home/alexander/code/dbuff-vite/node_modules/.pnpm/rollup@2.47.0/node_modules/rollup/dist/shared/rollup.js:5309:30)
    at Module.error (/home/alexander/code/dbuff-vite/node_modules/.pnpm/rollup@2.47.0/node_modules/rollup/dist/shared/rollup.js:9765:16)
    at Module.traceVariable (/home/alexander/code/dbuff-vite/node_modules/.pnpm/rollup@2.47.0/node_modules/rollup/dist/shared/rollup.js:10151:29)
    at ModuleScope.findVariable (/home/alexander/code/dbuff-vite/node_modules/.pnpm/rollup@2.47.0/node_modules/rollup/dist/shared/rollup.js:8905:39)
    at ChildScope.findVariable (/home/alexander/code/dbuff-vite/node_modules/.pnpm/rollup@2.47.0/node_modules/rollup/dist/shared/rollup.js:2636:38)
    at ClassBodyScope.findVariable (/home/alexander/code/dbuff-vite/node_modules/.pnpm/rollup@2.47.0/node_modules/rollup/dist/shared/rollup.js:2636:38)
    at FunctionScope.findVariable (/home/alexander/code/dbuff-vite/node_modules/.pnpm/rollup@2.47.0/node_modules/rollup/dist/shared/rollup.js:2636:38)
    at ChildScope.findVariable (/home/alexander/code/dbuff-vite/node_modules/.pnpm/rollup@2.47.0/node_modules/rollup/dist/shared/rollup.js:2636:38)
    at FunctionScope.findVariable (/home/alexander/code/dbuff-vite/node_modules/.pnpm/rollup@2.47.0/node_modules/rollup/dist/shared/rollup.js:2636:38)
    at ChildScope.findVariable (/home/alexander/code/dbuff-vite/node_modules/.pnpm/rollup@2.47.0/node_modules/rollup/dist/shared/rollup.js:2636:38)
 ERROR  Command failed with exit code 1.
```

</details>

**Comment by roberttolton** [comment]
@7architect Your logs say you're trying to import `nextTick` from `vue@2.6.12`..?

**Comment by iancallahan** [comment]
I am running into the same issue with Vite, Vue 3 and Rollup. Works fine with Vite dev and when I use the component elsewhere via Webpack. But Vite build yields the same issues as above.

**Comment by iancallahan** [comment]
I was able to get this to work with Rollup by forking it and simply updating the Vue version to ^3 and rebuilding.

**Comment by jljse** [comment]
I use vite + vue3 and have same trouble.
When I renamed node_modules/vue-masonry/node_modules folder, then it seems work.
Is this folder required?

**Comment by lahdekorpi** [comment]
`0.14.0-beta.0` Works perfectly. 🎉  Thank you!

**Comment by shershen08** [comment]
released `0.14.0` - https://github.com/shershen08/vue-masonry/releases/tag/0.14.0

https://www.npmjs.com/package/vue-masonry/v/0.14.0
